pkg/reconciler/taskrun/validate_taskrun_test.go

```diff
@@ -104,14 +103,17 @@ func TestValidateResolvedTask_ValidParams(t *testing.T) {
 			"key3": "val3",
 		}),
 	}}
-	m := []v1beta1.Param{{
-		Name:  "zoo",
-		Value: *v1beta1.NewStructuredValues("a", "b", "c"),
-	}}
-	if err := ValidateResolvedTask(ctx, p, &v1beta1.Matrix{Params: m}, rtr); err != nil {
+	m := &v1beta1.Matrix{
+		Include: []v1beta1.IncludeParams{{
+			Name: "build-1",
+			Params: v1beta1.Params{{
+				Name: "include", Value: v1beta1.ParamValue{Type: v1beta1.ParamTypeString, StringVal: "string-1"},
+			}},
+		}},
+	}
+	if err := ValidateResolvedTask(ctx, p, m, rtr); err != nil {
```

Comment:
this change seems to remove the test for m.Params, and add m.Include.Params -- please make sure that both are covered